wiki/Deprecation_policy.md

```diff
@@ -17,10 +17,9 @@ changes, named
 This is the current policy for deprecating and removing code from
 Biopython:
 
--   First, ask on the Biopython and biopython-dev [mailing
-    lists](Mailing_lists "wikilink") whether a given piece of code has
-    any users. Please keep in mind that not all users are following the
-    biopython-dev mailing list.
+-   First, ask on the [Biopython mailing lists](Mailing_lists "wikilink")
```

Comment:
--> list (singular)

Reply:
I'm sorry, I did not get this comment at the first sight. I have already commited the change. I'm not a native speaker.

Reply:
Even as a native speaker, I have committed a lots of typos - don't feel bad, this is one of the reasons for having pull request reviews.